[Bug]: Compiler error when WOLFSSL_RPIPICO is used.

### Contact Details

tjko@iki.fi

### Version

Latest (master)

### Description

Seem as if commit 30cb25e498d480c187d8a20d268bfbad65f8f516, broke compiling when using ```WOLFSSL_RPIPICO```.

```
wolfssl/wolfcrypt/src/port/rpi_pico/pico.c:45:16: error: 'BAD_FUNC_ARG' undeclared (first use in this function)
   45 |         return BAD_FUNC_ARG;
      |                ^~~~~~~~~~~~
```


### Reproduction steps

_No response_

### Relevant log output

```shell

```

--- a/wolfcrypt/src/port/rpi_pico/pico.c
+++ b/wolfcrypt/src/port/rpi_pico/pico.c
@@ -20,7 +20,7 @@
  */
 
 
-
+#include <wolfssl/wolfcrypt/libwolfssl_sources.h>
 
 #include <inttypes.h>
 #include <string.h>

